### PR TITLE
Avoids some unnecessary error suppression.

### PR DESCRIPTION
**Pull request by Maikuolan**

### CODE REVIEW

File: public_html/package-stats.php [hunk 1]
```diff
@@ -255,9 +255,9 @@ function updateGraph()
     }
 </script>
 <form name="graph_control" action="#">
- <input type="hidden" name="pid" value="<?php echo @$_GET['pid']; ?>" />
- <input type="hidden" name="rid" value="<?php echo @$_GET['rid']; ?>" />
- <input type="hidden" name="cid" value="<?php echo @$_GET['cid']; ?>" />
+ <input type="hidden" name="pid" value="<?php echo empty($_GET['pid']) ? '' : $_GET['pid']; ?>" />
```
Comment (petk) [inline public_html/package-stats.php]:
```suggestion
 <input type="hidden" name="pid" value="<?php echo isset($_GET['pid']) ? $_GET['pid'] : ''; ?>" />
```

File: public_html/package-stats.php [hunk 2]
```diff
@@ -255,9 +255,9 @@ function updateGraph()
...
+ <input type="hidden" name="rid" value="<?php echo empty($_GET['rid']) ? '' : $_GET['rid']; ?>" />
```
Comment (petk) [inline public_html/package-stats.php]:
```suggestion
 <input type="hidden" name="rid" value="<?php echo isset($_GET['rid']) ? $_GET['rid'] : ''; ?>" />
```

File: public_html/package-stats.php [hunk 3]
```diff
@@ -255,9 +255,9 @@ function updateGraph()
...
+ <input type="hidden" name="cid" value="<?php echo empty($_GET['cid']) ? '' : $_GET['cid']; ?>" />
```
Comment (petk) [inline public_html/package-stats.php]:
```suggestion
 <input type="hidden" name="cid" value="<?php echo isset($_GET['cid']) ? $_GET['cid'] : ''; ?>" />
```

Comment (petk) [reply public_html/package-stats.php]: Because when $_GET['someid'] = 0 it will be also empty and it won't output zero...

**Comment by php-pulls** [comment]
**Comment on behalf of petk at php.net:**

Applied via c3abde62a99a19a5fd13ab492ccc1273600fe1f7 thanks @Maikuolan :+1:

**Pull request closed without merging**